src/cmap/auth/mongodb_oidc/callback_workflow.ts

```diff
@@ -0,0 +1,209 @@
+import { type Document, Binary, BSON } from 'bson';
+import { promisify } from 'util';
+
+import { MongoInvalidArgumentError } from '../../../error';
+import { ns } from '../../../utils';
+import type { Connection } from '../../connection';
+import type { MongoCredentials } from '../mongo_credentials';
+import type { OIDCMechanismServerStep1, OIDCRequestTokenResult } from '../mongodb_oidc';
+import { AuthMechanism } from '../providers';
+import { TokenEntryCache } from './token_entry_cache';
+import type { Workflow } from './workflow';
+
+/* 5 minutes in milliseconds */
+const TIMEOUT = 300000;
```

Comment:
```suggestion
const TIMEOUT_MS = 300000;
```

maybe more clear in other places that this is a MS timeout?

Reply:
Updated.